build.sh

```diff
@@ -53,5 +53,5 @@ dockerbuild()
 if [ ! -z "$BUILD_IN_DOCKER" ]; then
     dockerbuild $args
 else
-    $DIR/run-build.sh $args
+    sudo $DIR/run-build.sh $args
```

Comment:
Probably merits a similar comment to the other file so folks know why you did it, possibly a link to the tracking issue so it can be reverted if/when this gets addressed by 1ES